stdlib/3/unittest/mock.pyi

```diff
@@ -101,7 +101,7 @@ class NonCallableMock(Base, Any):  # type: ignore
         def _get_call_signature_from_name(self, name: str) -> Any: ...
 
     def assert_any_call(self, *args: Any, **kwargs: Any) -> None: ...
-    def assert_has_calls(self, calls: _CallList, any_order: bool = ...) -> None: ...
+    def assert_has_calls(self, calls: List[_Call], any_order: bool = ...) -> None: ...
```

Comment:
Could this be any sequence or iterable?

Reply:
Hm, good question. I have seen only lists internally, but I will check the source and update to wider type if possible.

Reply:
It looks like an arbitrary iterable will work.

Reply:
Hm, but actually `any_order` argument suggests it logically should be something ordered (even though the method doesn't actually need this). Probably `Sequence` is better, so that people will not pass a set there and be surprised.